Added totp retrieval

jihbuomj opened the pull request:
Implementation to resolve #20.

Significantly modifies main code, so I am open to suggestions.
Also unsure of what tests I should be running other than the shellcheck.

yardnsm reviewed (changes requested):
Awesome, it looks amazing! 😄  Only thing missing is a nice header to remind us about this feature. Something like https://github.com/rafi/tmux-pass/blob/master/scripts/main.sh#L84.

Regarding the tests, shellcheck will suffice.

jihbuomj commented:
Added the header and I moved the arguments to a variable to avoid using export. Also wasn't sure if that environment variable overwrote any intended default behavior.

yardnsm commented:
Amazing, thank you!
